introduction.md

```diff
@@ -0,0 +1,3 @@
+Hi! My name's Alex and I live in north north Austin, Texas. I love learning everything I can about code!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.